examples/apps/quick_tap.py

```diff
@@ -0,0 +1,466 @@
+        print('You won {} round{}'.format(self.player.score, 's' if self.player.score != 1 else ''))
+        print('Cozmo won {} round{}'.format(self.cozmo_player.score, 's' if self.cozmo_player.score != 1 else ''))
+        if self.cozmo_player.score > self.player.score:
+            print('~COZMO WINS QUICK TAP~')
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapGameCozmoWinHighIntensity).wait_for_completed()
+        else:
+            print('~PLAYER WINS QUICK TAP~')
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapGamePlayerWinHighIntensity).wait_for_completed()
+
+
+class QuickTapPlayer():
+    '''Player-specifc Quick Tap logic.'''
+    def __init__(self):
+        self.cube = None
+        self.score = 0
+        self.has_tapped = False
+        self.name = 'Player'
+        self.elapsed_tap_time = None
+        self.won_round = False
+
+    def wins_round(self):
+        '''Prints winning message, updates score, and sets won_round flag to True.'''
+        print('****{} wins the round****'.format(self.name))
+        self.score += 1
+        self.won_round = True
+
+    def reset(self):
+        '''Resets elapsed_tap_time, and sets has_tapped and won_round flags to False.'''
+        self.elapsed_tap_time = sys.maxsize
+        self.has_tapped = False
+        self.won_round = False
+
+    def register_tap(self, round_start_time):
+        '''Calculates elapsed time of tap, and sets has_tapped flag to True.
+
+        Args:
+            round_start_time (Time): time stamp set in QuickTapGame to calculate players' tap_times
+        '''
+        self.elapsed_tap_time = time.time() - round_start_time
+        self.has_tapped = True
+
+
+class CozmoQuickTapPlayer(QuickTapPlayer):
+    '''Cozmo-specific Quick Tap player logic, with a reference to the actual Cozmo robot.
+        
+    Args:
+        robot (cozmo.robot.Robot): passed in from the QuickTapGame class
+    '''
+    def __init__(self, robot: cozmo.robot.Robot):
+        super().__init__()
+        self.robot = robot
+        self.name = 'Cozmo'
+
+    async def select_cube(self):
+        '''Cozmo looks for a cube, drives to it, and taps it.'''
+        self.cube = await self.robot.world.wait_for_observed_light_cube()
+        self.cube.set_lights(cozmo.lights.white_light)
+        await asyncio.sleep(2)
+        self.cube.start_light_chaser(0.5)
+        await self.robot.set_lift_height(1.0).wait_for_completed()
+        await self.robot.go_to_object(self.cube, distance_mm(40)).wait_for_completed()
+        await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapTap).wait_for_completed()
+        self.cube.stop_light_chaser()
+        self.cube.set_lights(green_light)
+
+    async def determine_move(self, buzzer_display_type):
+        '''Cozmo chooses a move based on the probabilities above.
+
+        Args:
+            buzzer_display_time (string): the display of the buzzers, either RED, SAME, or DIFFERENT
```

Comment:
buzzer_display_time should be buzzer_display_type

"RED, SAME or DIFFERENT" should be "MAKE_BUZZERS_RED, MAKE_BUZZERS_SAME_COLORS or MAKE_BUZZERS_DIFFERENT_COLORS"